corehq/apps/app_manager/models.py

```diff
@@ -3443,13 +3445,15 @@ def __init__(self, *args, **kwargs):
 
     @classmethod
     def wrap(cls, doc):
-        # for backwards compatibility with apps that have localized descriptions
-        from corehq.apps.userreports.util import default_language, localize
-        if isinstance(doc.get('description'), dict):
-            if doc['description']:
-                doc['description'] = localize(doc['description'], default_language())
-            else:
-                doc['description'] = ''
+        # for backwards compatibility with apps that have localized or xpath descriptions
```

Comment:
I don't think it should be too complex - agree that this lazy migration code shouldn't hang around indefinitely.